src/test/java/org/wiremock/RandomHelperTest.java

```diff
@@ -0,0 +1,43 @@
+package org.wiremock;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.github.tomakehurst.wiremock.extension.responsetemplating.helpers.HandlebarsHelperTestBase;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.Random;
+import net.datafaker.Faker;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class RandomHelperTest extends HandlebarsHelperTestBase {
+  RandomHelper helper;
+
+  @BeforeEach
+  public void init() {
+    helper = new RandomHelper();
+  }
+
+  @Test
+  public void rendersAMeaningfulErrorWhenExpressionIsInvalid() throws IOException {
+    testHelperError(
+        helper,
+        "something really random",
+        null,
+        is("[ERROR: Unable to evaluate the expression something really random]"));
+  }
+
+  @ParameterizedTest
+  @CsvSource(value = {"123456789, Name.firstName, Herb", "123456789, Name.lastName, Hauck"})
+  public void returnsRandomValue(int seed, String expression, String expected) throws Exception {
+    Field newFaker = helper.getClass().getDeclaredField("faker");
```

Comment:
How about we allow the seed to be optionally passed to the constructor of `RandomHelper`? That way it's easier to set up the test and there's a hook for users to set their own seed if they prefer.

Reply:
Yes, I had considered that but maybe it would be better to hold off making changes to the API till the seed discussion from https://github.com/wiremock/wiremock/issues/2412 is finalised?